README.md

```diff
@@ -66,12 +66,18 @@ Lo primero que debes crear es el objeto de requerimiento para el SDK el cual se
 
 ````javascript 1.5
 var options = {
-  endpoint: './transaction-create',
-  commerceLogo: '/onepay-sdk-example/images/icons/logo-01.png',
-  callbackUrl: './transaction-commit'
+  endpoint: '//tu-url.com/transaction-create',
+  commerceLogo: '//tu-url.com/images/icons/logo-01.png',
+  callbackUrl: '//tu-url.com/transaction-commit'
 };
 ````
 
+Debido a que el modal reside en un dominio fura de tu comercio, debes usar URLs completas para que los callbacks
+funcionen como se espera.
```

Comment:
I think this is no longer true. Only the commer Logo url is used inside the iframe. The other two are used on the "commerce" component and therefore can be relative, I think.

Reply:
Logo and callbackUrl you need to put complete URL if you want the use case MOBILE works fine to callback.

Reply:
But for MOBILE the callbackUrl is provided by the backend and it doesn't matter what they put in the javascript JS. 

So it's still only for the logo.

Reply:
Im sorry you are right, this callback is used only for WEB use case. In MOBILE use the callback setted in backed. I have fixed readme.